### DISCUSSION TITLE
What is cloners ar in YHYPE?????

### DISCUSSION THREAD

**Discussion by athrvvvv** [yhype/yhype-feedback#9] [Q&A] [answered]
What is cloners ar in YHYPE????? and what is clones in YHYPE?
CLONERS vs CLONES
[You can see here](https://yhype.me/github/accounts/athrvvvv/custom_cmd/downloads)

**Comment by antonkomarev** [comment, marked as answer]
- Cloners — the number of people (robots) who downloaded your sources.
- Clones — number of downloads of the source code.

If I will download your repository source code 3 times, you will have +1 cloner and +3 clones.
